helm/dashboards/charts/private_dashboards_mz/dashboards/shared/private/zot.json

```diff
@@ -0,0 +1,3472 @@
+          ]
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "mimir"
+        },
+        "definition": "label_values(zot_scheduler_workers_tasks_duration_seconds_bucket,name)",
+        "description": "zot_http_method_latency_seconds_bucket method",
+        "hide": 2,
+        "includeAll": true,
+        "multi": true,
+        "name": "scheduler_workers_tasks_duration",
+        "options": [],
+        "query": {
+          "query": "label_values(zot_scheduler_workers_tasks_duration_seconds_bucket,name)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 1,
+        "type": "query"
+      },
+      {
+        "current": {
+          "selected": true,
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "mimir"
+        },
+        "definition": "label_values(zot_storage_lock_latency_seconds_bucket,storageName)",
+        "description": "zot_storage_lock_latency_seconds_bucket",
+        "hide": 2,
+        "includeAll": true,
+        "multi": true,
+        "name": "storageName",
+        "options": [],
+        "query": {
+          "query": "label_values(zot_storage_lock_latency_seconds_bucket,storageName)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 1,
+        "type": "query"
+      },
+      {
+        "current": {
+          "selected": true,
+          "text": [
+            "giantswarm"
+          ],
+          "value": [
+            "giantswarm"
+          ]
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "mimir"
+        },
+        "definition": "label_values(zot_repo_downloads_total,repo)",
+        "description": "For repository names separated by slash, this is the part before the first separator.",
+        "hide": 0,
+        "includeAll": true,
+        "label": "Repository prefix",
+        "multi": true,
+        "name": "repo_prefix",
+        "options": [],
+        "query": {
+          "query": "label_values(zot_repo_downloads_total,repo)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 1,
+        "regex": "([a-z-.]+)/(.*)",
+        "skipUrlSync": false,
+        "sort": 1,
+        "type": "query"
+      },
+      {
+        "description": "Name of the PersistentVolumeClaim for the container image storage",
+        "hide": 2,
+        "name": "zot_pvc_name",
+        "query": "flux-giantswarm-zot-pvc",
+        "skipUrlSync": false,
+        "type": "constant"
+      }
+    ]
+  },
+  "time": {
+    "from": "now-2d",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "utc",
+  "title": "In-cluster container registry (Zot)",
+  "uid": "2JZeZ6hSk",
```

Comment:
You may want to set a fancy UID so it's easier to link it (from alerts for instance).

Reply:
Something like `zot-container-registry`?

Reply:
Something like that yes.